lib/ops/opsservice/configure.go

```diff
@@ -599,7 +601,9 @@ func (s *site) getPlanetMasterSecretsPackage(ctx *operationContext, p planetMast
 		return nil, trace.Wrap(err)
 	}
 
-	baseKeyPair, err := archive.GetKeyPair(constants.APIServerKeyPair)
+	// Don't rotate apiserver secrets, as this secret is currently used to authenticate service account tokens
+	// TODO(securty) support rotation of apiserver / serviceaccount secrets
```

Comment:
In 1.17 they rotate off disk if you put it down that way, ever hour iirc if this is for k8s apiserver